code/datums/skills/skating.dm

```diff
@@ -0,0 +1,4 @@
+/datum/skill/skating
+	name = "Skating"
+	desc = "IF YOU CAN TWEAK A PRESSURE FLIP WHILE BOMBING A HILL, YOU'RE ONE GNAR TOO FAR BROTHER"
+	//modifiers = list(SKILL_SPEED_MODIFIER = list(1, 0.9, 0.8, 0.7, 0.6, 0.5, 0.36)) //speed also touches probability in using up a soap's charge
```

Comment:
Is this comment still supposed to be here? It refers to soap...

Reply:
Ya gotta be clean to do these slams bro